analyses/focal-cn-file-preparation/01-add-ploidy-cnvkit.Rmd

```diff
@@ -17,19 +17,42 @@ library(dplyr)
 ### Read in data
 
 ```{r}
-cnvkit_file <- file.path("..", "..", "data", "cnv-cnvkit.seg.gz")
+cnvkit_file <- file.path("..", "..", "data", "v8", "cnv-cnvkit.seg.gz")
 cnvkit_df <- readr::read_tsv(cnvkit_file)
 ```
 
 ```{r}
-histologies_file <- file.path("..", "..", "data", "histologies.tsv")
+histologies_file <- file.path("..", "..", "data", "v8", "histologies.tsv")
 histologies_df <- readr::read_tsv(histologies_file, guess_max = 100000)
 ```
 
+
+# Edits for https://github.com/PediatricOpenTargets/ticket-tracker/issues/177
+# For PBTA, histologies file has germline_sex_estimate already assigned to WXS patients which have had WGS performed
+# Updating here germline_sex_estimate for cohort=="TARGET" where is.na(germline_sex_estimate)
+#Isolate IDs where germline_sex_estimate is NA and cohort == "TARGET"
```

Comment:
see my comment below about the cohort selection

Reply:
The code now uses sample_type in combination with experimental_strategy, instead of specific cohort selection.